spec.yaml

```diff
@@ -139,16 +141,37 @@ paths:
             Retry-After:
               description: >
                 Indication for the client when the service could be requested
-                again in HTTP Date format as used by the 
-
-                Internet Message Format [RFC5322] (e.g. `Wed, 21 Oct 2015
-                07:28:00 GMT`) or the number of seconds 
-
-                (e.g. 3000 if you the service is expected to be available again
-                within 50 minutes).
+                again in HTTP Date format as used by the Internet Message Format
+                [RFC5322] (e.g. `Wed, 21 Oct 2015 07:28:00 GMT`) or the number
+                of seconds (e.g. 3000 if you the service is expected to be
+                available again within 50 minutes).
               type: string
           schema:
             $ref: '#/definitions/Error'
+  /invite-accepted:
+    post:
+      summary: Inform the sender that the invitation was accepted to start sharing.
+      description: Inform about an accepted invitation so the user on the sender provider's side can initiate the OCM share creation.
+      parameters:
+        - name: invite
```

Comment:
IIUC you're proposing two deviations from the current implementation as it exists in Reva:
1. change the endpoint from e.g. `https://sender.com/iop/ocm/invite/accept` to e.g. `https://sender.com/iop/ocm/invite-accepted` -> as agreed, that makes it clearer
2. change the JSON from:
```
{
  "token": "10431cd1-11df-49ce-af87-9dfc",
  "recipientProvider": "surf-iop-rd-app-acc.data.surfsara.nl",
  "userID": "marie",
  "name": "Marie Curie",
  "email":  "marie@surfsara.nl"
}
```
to (IIUC):
```
{
  "invite" : {
    "token": "10431cd1-11df-49ce-af87-9dfc",
    "recipientProvider": "surf-iop-rd-app-acc.data.surfsara.nl",
    "userID": "marie",
    "name": "Marie Curie",
    "email":  "marie@surfsara.nl"
  }
}
```
Can this second deviation be avoided? Does it serve a purpose?

Reply:
This was also not intended and comes from the original PR. Will fix it.

Reply:
As discussed on Gitter, see https://swagger.io/docs/specification/2-0/describing-request-body/ - 
> The payload name. It is required but ignored (it is used for documentation purposes only).

So the existing current format for `/shares` is (confusing but) correct, and so is yours here!

Reply:
Sorry for the confusion, you don't need to change anything.